provision-osx/tasks/configure_base_system.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+-
+  stat:
+    path: ~/.bash_profile
+  register: bash_file
+
+-
+  name: Create user bash file
+  file:
+    path: ~/.bash_profile
+    state: touch
+  when: bash_file.stat.exists == False
+
+-
+  name: Configure bash profile language encoding
+  lineinfile:
+    dest: ~/.bash_profile
+    line: "export LANG={{ buildfarm_lang_env_var }}"
+
+-
+  name: Configure profile file
+  lineinfile:
+    dest: ~/.profile
+    line: "source ~/.bash_profile"
```

Comment:
@odra looks like this line is causing a problem on the Mac server.  There is already a `.bash_profile` on the Mac server and it has this line:

```
[[ -s "$HOME/.profile" ]] && source "$HOME/.profile" # Load the default .profile
```

But here the `.profile` is referencing the `.bash_profile` again. It is causing circular dependency between the 2 files and the ssh login doesn't work anymore.

Is this line required? If not, might be better remove it? I think by default, all `.profile`, `.bash_profile` and `.bashrc` are all loaded anyway?